orchestration/src/test/java/com/sap/ai/sdk/orchestration/OrchestrationUnitTest.java

```diff
@@ -316,9 +316,7 @@ void maskingAnonymization() throws IOException {
     assertThat(inputMasking.getMessage()).isEqualTo("Input to LLM is masked successfully.");
     assertThat(inputMasking.getData()).isNotNull();
     final var choices = ((LLMModuleResultSynchronous) result.getOrchestrationResult()).getChoices();
-    assertThat(choices.get(0).getMessage().getContent())
-        .isEqualTo(
-            "I'm sorry, I cannot provide information about specific individuals, including their nationality.");
+    assertThat(choices.get(0).getMessage().getContent()).contains("Hi Mallory");
```

Comment:
(Comment)

This is the fix.

Reply:
Sorry, I didn't realize the refactoring buried the actual fix. This is just a driveby from using different test data. The actual fix is here: https://github.com/SAP/ai-sdk-java/pull/165/files#diff-7a166e1d85cf8eada0b2f93810ca498b836225f4ee408cb16718b4d508e5264aR19